### PR TITLE
OPT: Using github actions instead of travis-ci

### PR DESCRIPTION
**Pull request by ricardosantosalves**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: Run tests for timeexecution
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9, '3.10']
```
Comment (kammala) [inline .github/workflows/test.yml:10]: looks inconsistent that only `3.10` is surrounded with quotes

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: Lint checks for timeexecution
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
```
Comment (kammala) [inline .github/workflows/lint.yml:10]: just curious, why do we use the oldest supported version for linting?

File: setup.cfg [hunk 3]
```diff
@@ -11,5 +11,8 @@ ignore =
     # mutual exclusive with W504
     W503
 
-[mypy-influxdb.*,elasticsearch.*,fqn_decorators.*,pkgsettings.*,kafka.*,setuptools.*,Queue.*]
+[mypy]
+ignore_missing_imports = true
+
+[mypy-mock,freezegun]
```
Comment (kammala) [inline setup.cfg:17]: FYI, `mypy` now supports `pyprohect.toml`

and it looks better when placed there:
```
[tool.mypy]
ignore_missing_imports = true

[[tool.mypy.overrides]]
module = [
  "mock",
  "freezegun",
]
ignore_missing_imports = true
```

BTW, why do we need to enable `ignore_missing_imports` for both global mypy configuration and these two modules? shouldn't it be enough to have only global one?

File: .github/workflows/lint.yml (hunk 2 above)
Comment (igbun) [reply .github/workflows/lint.yml:10]: Just minimizing changes for now. Will change this one later.

File: .github/workflows/test.yml (hunk 1 above)
Comment (igbun) [reply .github/workflows/test.yml]: Indeed. Fixed.

File: setup.cfg (hunk 3 above)
Comment (igbun) [reply setup.cfg]: Good one. Looks better now I think.

**Review by kammala — APPROVED** [review]

**Review by kammala — APPROVED** [review]

**Review by beerdeaap — APPROVED** [review]

**Review by beerdeaap — APPROVED** [review]

**Pull request merged**